assets/src/hooks/useVehicles.ts

```diff
@@ -46,6 +56,8 @@ interface VehicleData {
   schedule_adherence_secs: number
   schedule_adherence_string: string
   scheduled_headway_secs: number
+  sources: string[]
```

Comment:
Is this field used at all? Each discrepency has its own list of sources already.